app/models/visit.rb

```diff
@@ -9,12 +9,26 @@ class Visit < ActiveRecord::Base
     :processing_state,
     presence: true
 
-  delegate :email_address, to: :prison, prefix: true
+  validates :email_override,
+    inclusion: %w[ bounced spam_reported],
+    allow_nil: true
+
+  delegate :email_address, :phone_no, :name, to: :prison, prefix: true
+  alias_attribute :recipient, :visitor_email_address
+  alias_attribute :first_date, :slot_option_1
+
+  def total_number_of_visitors
+    additional_visitors.count + 1
+  end
 
   def prisoner_full_name
     [prisoner_first_name, prisoner_last_name].join(' ')
   end
 
+  def prisoner_last_name_initial
```

Comment:
do we ever use this in a context that isn't something like "Paul B"? If that's the case, it might be more intentional to have something like `prisoner_anonymized_name` instead of appending the initial in views.